app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -354,14 +358,19 @@ public int getFeedCounterSum() {
     };
 
     private void loadData() {
+        progressBar.setVisibility(View.VISIBLE);
         disposable = Observable.fromCallable(DBReader::getNavDrawerData)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(result -> {
                     navDrawerData = result;
                     updateSelection(); // Selected item might be a feed
                     navAdapter.notifyDataSetChanged();
-                }, error -> Log.e(TAG, Log.getStackTraceString(error)));
+                    progressBar.setVisibility(View.GONE); },
+                error -> {
```

Comment:
Sorry @Xeitor, looks like I was wrong. Checkstyle still complains. I experimented a bit and it looks like the following actually works :D

```
                .subscribe(
                    result -> {
                        navDrawerData = result;
                        updateSelection(); // Selected item might be a feed
                        navAdapter.notifyDataSetChanged();
                        progressBar.setVisibility(View.GONE);
                    }, error -> {
                        Log.e(TAG, Log.getStackTraceString(error));
                        progressBar.setVisibility(View.GONE);
                    });
```

(add a line break for the lambda parameter)

Reply:
Okay, don't worry. Pushing the changes right away. Whay do you mean when you say that chestyle complains?

Reply:
I mean the tests below. The tool "checkstyle" (static analysis) ensures that the code style is consistent throughout the project. It is executed for every pull request. Pull requests are usually only merged if all tests pass (PR gets a green check mark). You can click on failed tests to see details about why they fail.

Reply:
Nice thanks for the info. I pushed the changes some minutes ago, we are good to go right?